control/tests/freqresp_test.py

```diff
@@ -236,7 +236,7 @@ def test_options(self):
 
 
 def suite():
-   return unittest.TestLoader().loadTestsFromTestCase(TestTimeresp)
+   return unittest.TestLoader().loadTestsFromTestCase(TestFreqresp)
 
 if __name__ == '__main__':
    unittest.main()
```

Comment:
This is code from the unittest framework. Pytest does not use it. IMHO, let's remove `suite()` and the `__main__` sections from all test files.

Reply:
I have removed the `suite()` functions, which were often not properly set up and were only used in `tests/run_all.py`, which I don't think we were using anyway.

I left in the `__main__` sections, so that you could (if you wanted) call a test script directly (rather than having to use `pytest`).